docs/release-notes/self-hosted/self-hosted-v12.0.0.md

```diff
@@ -0,0 +1,161 @@
+    -   [<span class="skip-vale">eslint-plugin-tailwindcss</span>](https://www.npmjs.com/package/eslint-plugin-tailwindcss) (TS-419)
+    -   [<span class="skip-vale">eslint-plugin-test-selectors</span>](https://www.npmjs.com/package/eslint-plugin-test-selectors) (TS-419)
+    -   [<span class="skip-vale">eslint-plugin-you-dont-need-lodash-underscore</span>](https://www.npmjs.com/package/eslint-plugin-you-dont-need-lodash-underscore) (TS-419)
+    -   [<span class="skip-vale">eslint-plugin-filenames</span>](https://www.npmjs.com/package/eslint-plugin-filenames) (TS-424)
+    -   [<span class="skip-vale">eslint-plugin-diff</span>](https://www.npmjs.com/package/eslint-plugin-diff) (TS-428)
+    -   [<span class="skip-vale">eslint-config-airbnb-typescript-prettier</span>](https://www.npmjs.com/package/eslint-config-airbnb-typescript-prettier) (TS-448)
+    -   [<span class="skip-vale">eslint-plugin-jest-dom</span>](https://www.npmjs.com/package/eslint-plugin-jest-dom) (TS-449)
+    -   [<span class="skip-vale">eslint-plugin-testing-library</span>](https://www.npmjs.com/package/eslint-plugin-testing-library) (TS-449)
+    -   [<span class="skip-vale">eslint-plugin-sort-destructure-keys</span>](https://www.npmjs.com/package/eslint-plugin-sort-destructure-keys) (TS-448)
+-   Improved the performance of the tool Stylelint by reviewing and refactoring the tool configurations and supported packages. (TS-438)
+-   Updated and refactored the tool Ameba. [Thanks to <span class="skip-vale">Sija</span> for the contribution!](https://github.com/codacy/codacy-ameba/pull/25) (TS-417)
+
+## Tool versions
+
+This version of Codacy Self-hosted includes the tool versions below. The tools that were updated on this version are highlighted in bold:
+
+-   **[Ameba 1.4.3](https://github.com/crystal-ameba/ameba/releases/tag/v1.4.3) (updated from 0.13.1)**
+-   Bandit 1.7.0
+-   Brakeman 4.3.1
+-   bundler-audit 0.6.1
+-   **[Checkov 2.3.187](https://github.com/bridgecrewio/checkov/releases/tag/2.3.187) (updated from 2.1.188)**
+-   Checkstyle 10.3.1
+-   Clang-Tidy 10.0.1
+-   CodeNarc 3.2.0
+-   CoffeeLint 2.1.0
+-   **[Cppcheck 2.10.3](https://github.com/danmar/cppcheck/releases/tag/2.10.3) (updated from 2.10)**
+-   Credo 1.4.0
+-   CSSLint 1.0.5
+-   dartanalyzer 2.17.0
+-   **[detekt 1.22.0](https://github.com/detekt/detekt/releases/tag/v1.22.0) (updated from 1.19.0)**
+-   **[ESLint 8.44.0](https://github.com/eslint/eslint/releases/tag/v8.44.0) (updated from 8.34.0)**
+-   ESLint (deprecated) 7.32.0
+-   Faux-Pas 1.7.2
+-   Flawfinder 2.0.19
+-   **[Gosec 2.15.0](https://github.com/securego/gosec/releases/tag/v2.15.0) (updated from 2.8.1)**
+-   Hadolint 1.18.2
+-   Jackson Linter 2.14.2
+-   JSHint 2.13.5
+-   markdownlint 0.26.2
+-   PHP Mess Detector 2.13.0
+-   PHP_CodeSniffer 3.7.2
+-   PMD 6.55.0
+-   **[Prospector 1.10.2](https://github.com/PyCQA/prospector/releases/tag/v1.10.2) (updated from 1.7.7)**
```

Comment:
Fixes the broken link:

```suggestion
-   **[Prospector 1.10.2](https://github.com/landscapeio/prospector/releases/tag/1.10.2) (updated from 1.7.7)**
```

Reply:
I already opened a separate pull request to review the URLs across the other documentation pages: https://github.com/codacy/docs/pull/1789